code/modules/mod/mod_types.dm

```diff
@@ -53,10 +53,13 @@
 		/obj/item/mod/module/welding,
 		/obj/item/mod/module/rad_protection,
 		/obj/item/mod/module/flashlight,
+		/obj/item/mod/module/tether,
 		/obj/item/mod/module/magboot,
 	)
 	default_pins = list(
 		/obj/item/mod/module/magboot,
+		/obj/item/mod/module/flashlight,
```

Comment:
go in module order, flashlight would be first
also shouldnt chief engi have it pinned too?

Reply:
yeah I'll sort them all to be consistent